source/stylesheets/refills/_footer-2.scss

```diff
@@ -111,17 +93,18 @@
     }
 
     ul.footer-social {
-      margin-top: 1em;
+      margin: $small-spacing 0 0 0;
 
       @include media($large-screen) {
         float: right;
-        margin-top: 0;
+        margin: 0 0 0 $base-spacing;
       }
 
       li {
-        font-size: 1em;
+        font-size: $base-font-size;
```

Comment:
Properties should be ordered float, font-size, line-height, margin, padding-right